### DISCUSSION TITLE
Multiple windows diferent contexts, it is corret this way?

### DISCUSSION THREAD

**Discussion by adarosci** [wailsapp/wails#3610] [General]
This is an example, I have 3 windows (main, logs and alert)

![image](https://github.com/user-attachments/assets/8bd4744f-4175-464d-87dc-654dadf36b55)

I create the main window and after startup I create logs and after startup logs I create alert

![image](https://github.com/user-attachments/assets/1ce9d286-a655-4ee6-8a90-e7fbba3d128f)

![image](https://github.com/user-attachments/assets/232c269c-147c-49e2-835f-d23ed38ce350)

![image](https://github.com/user-attachments/assets/8b60e597-a531-4d29-a4a4-b70bfd9a27bb)

![image](https://github.com/user-attachments/assets/22c9e2e7-f446-4dfc-aaf2-932cb1741cad)

This way was the form I found to work this but I think is wrong.

Today I have a application in Electron that I have so many problems because I execute an application in background with Golang, with Wails I can all in only application.

You can tell me if is correct or no? and if no way do I make correct this.

Grat

**Comment by leaanthony** [comment]
Multi window is not supported in v2 but we do support it in v3 which is currently alpha. For more info: https://v3alpha.wails.io
